### ISSUE TITLE
Training convergence speed?

### ISSUE THREAD

**Issue by fx-hit** [PKU-HMI-Lab/Hybrid-VLA#11]
We started training based on the OpenVLA checkpoint, using the Bridge V2 dataset. After 30,000 iterations, the loss only decreased from over 3 to 2.7. Is this a normal speed? If we use the pre-trained checkpoint from Hybrid-VLA, the training loss starts around 1 and quickly drops below 0.7.
